src/dsp_tools/utils/xmlupload/write_diagnostic_info.py

```diff
@@ -3,117 +3,54 @@
 import json
 import os
 from collections import namedtuple
-from datetime import datetime
 from pathlib import Path
-from typing import Any, Optional, Union
+from typing import Any
 
 import pandas as pd
-import regex
 from lxml import etree
 
 from dsp_tools.utils.create_logger import get_logger
+from dsp_tools.utils.xmlupload.upload_config import UploadConfig
 
 MetricRecord = namedtuple("MetricRecord", ["res_id", "filetype", "filesize_mb", "event", "duration_ms", "mb_per_sec"])
```

Comment:
@Nora-Olivia-Ammann @BalduinLandolt Not the topic of this PR, but just FYI: We once added some metrics to the xmlupload, which I guess are now redundant, because you are going to implement better measurements. The metrics we implemented back then were made "by hand", i.e. there are a lot of lines all over the code doing time measurements. If you want, you can remove them.